zetaclient/orchestrator/orchestrator.go

```diff
@@ -116,9 +116,6 @@ func (oc *Orchestrator) MonitorCore(ctx context.Context) error {
 	shutdownOrchestrator := func() {
 		// now stop orchestrator and all observers
 		close(oc.stop)
-		for _, c := range oc.observerMap {
-			c.Stop()
```

Comment:
Could you explain this change?

Reply:
the observervers are already being stopped here 
https://github.com/zeta-chain/zeta-node/blob/59515eb811ab940564b336cf08a5ac328986ebeb/cmd/zetaclientd/start.go#L356-L358

Reply:
https://github.com/zeta-chain/node/pull/2538#discussion_r1689904775